app/jobs/scheduled/update_topic_hot_scores.rb

```diff
@@ -4,8 +4,17 @@ module Jobs
   class UpdateTopicHotScores < ::Jobs::Scheduled
     every 10.minutes
```

Comment:
I don't think we need to run this job every 10 minutes when we only recalculate this only every 6 hours?

Reply:
every 10 minutes if hot is enabled ... every 6 hours if hot is not in top menu... just saving on some sql work for people without hot.

Reply:
Ahh ok sorry I misread the conditional.